src/phasespace/load_mocap/load_mocap.py

```diff
@@ -16,16 +16,15 @@
 import scipy as sp
 import scipy.linalg as spla
 import numpy.linalg as la
-from mpl_toolkits.mplot3d import Axes3D
```

Comment:
We actually do need this import for MocapSource.plot_frame() to work, even though the Axes3D name doesn't show up anywhere else in the file (just because of matplotlib's general weirdness).